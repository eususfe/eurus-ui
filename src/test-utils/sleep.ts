/*
 * @Author: jiangxd
 * @Date: 2022-09-23 14:06:55
 * @LastEditTime: 2022-09-23 14:07:04
 * @LastEditors: jiangxd
 * @Description:
 * @FilePath: /eurus-ui/src/test-utils/sleep.ts
 */
const sleep = (time = 0) => new Promise(resolve => setTimeout(resolve, time));

export default sleep;
