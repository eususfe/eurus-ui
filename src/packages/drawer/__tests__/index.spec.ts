import { mount } from '@vue/test-utils';
import EDrawer from '..';

describe('eDrawer', () => {
	it('eDrawer snapshot', () => {
		const wrapper = mount(EDrawer);
		expect(wrapper.html()).toMatchSnapshot();
		wrapper.unmount();
	});
});
