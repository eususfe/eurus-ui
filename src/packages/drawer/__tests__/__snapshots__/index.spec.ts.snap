// Vitest Snapshot v1

exports[`EDrawer > EDrawer snapshot 1`] = `
"<transition-stub name=\\"EuDrawer\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\">
  <!---->
</transition-stub>"
`;
